frontend/src/components/dashboard/ImgMediaCard.js

```diff
@@ -1,48 +1,46 @@
 import React from 'react';
 import {makeStyles} from '@material-ui/core/styles';
+import {Link} from 'react-router-dom';
 import Card from '@material-ui/core/Card';
 import CardActionArea from '@material-ui/core/CardActionArea';
 import CardActions from '@material-ui/core/CardActions';
 import CardContent from '@material-ui/core/CardContent';
 import CardMedia from '@material-ui/core/CardMedia';
 import Button from '@material-ui/core/Button';
 import Typography from '@material-ui/core/Typography';
-import PlaceholderImage from './images/placeholder-image.png';
+import PlaceholderImage from './images/paint.jpg';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) =>({
   root: {
     maxWidth: 345,
   },
-});
+  link: {
+    textDecoration: 'none',
+    color: theme.palette.text.primary,
+  },
+}));
 
-export default function ImgMediaCard() {
+export default function ImgMediaCard({name, link}) {
   const classes = useStyles();
 
   return (
     <Card className={classes.root}>
-      <CardActionArea>
-        <CardMedia
-          component="img"
-          alt="artwork"
-          height="140"
-          image={PlaceholderImage}
-          title="Artwork"
-        />
-        <CardContent>
-          <Typography gutterBottom variant="h5" component="h2">
-            Artwork
-          </Typography>
-          <Typography variant="body2" color="textSecondary" component="p">
-            Lizards are a widespread group of squamate reptiles, with over
-            6,000 species, ranging across all continents except Antarctica
-          </Typography>
-        </CardContent>
-      </CardActionArea>
-      <CardActions>
-        <Button size="small" color="primary">
-          Learn More
-        </Button>
-      </CardActions>
+      <Link to={link} className={classes.link}>
+        <CardActionArea>
+          <CardMedia
+            component="img"
+            alt="artwork"
+            height="200"
+            image={PlaceholderImage}
+            title="Artwork"
+          />
+          <CardContent className ={classes.name}>
```

Comment:
smallest nit: spacing

Reply:
Just to double check, you are referring to this: `className={classes.name}` right?